script.js

```diff
@@ -1,4 +1,312 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+var deck = [];
+var playerHand = [];
+var computerHand = [];
+var playerScore;
+var computerScore;
+var displayPlayerHand = [];
+var displayComputerHand = [];
+var winner = "";
+var input;
+
+// Draw a card
+var drawCard = function (numCards, hand) {
+  for (var i = 0; i < numCards; i += 1) {
+    var newCard = deck.pop();
+    hand.push(newCard);
+  }
+  return newCard;
+};
+
+// Display card
+var displayCard = function (hand) {
```

Comment:
I would expect the displayCard function to return me a nicely formatted string that shows the details of each card but it seems that you return an array with the name of each card? Not too sure if this function is helpful